### PR TITLE
PD editor add pins

### PR DESCRIPTION
**Pull request by VelislavaC4Nexus**
### All pull requests:
* [ ] Have you considered security (e.g., XSS)?
* [ ] Have you considered desktop, mobile, and tablet form-factors?
* [ ] Have you considered [accessibility best practices](https://www.w3.org/WAI/standards-guidelines/wcag/)?

### Code
* [ ] Are the commits squashed into one commit?
* [ ] Is the [branch name and commit message](CONTRIBUTING.md#conventions-for-branch-names-and-commit-messages) following team convention?
* [ ] Is the code linted?
* [ ] Are all open issues, questions, and concerns by reviewers answered, resolved, and reviewed?

### Quality assurance
* [ ] If applicable, are unit tests written and are they passing?
* [ ] If applicable, are integration tests written and are they passing?
* [ ] Have checks linked in your pull request passed?

### Documentation
* [ ] Are there meaningful code comments included?

## Applicable to community contributors:
* [ ] Have you granted SFRA team access to your fork? [Grant access](CONTRIBUTING.md#community-contributors) 

### Documentation
* [ ] If applicable, is the change log updated?
* [ ] If applicable, have any UI text changes been reviewed by Documentation?
* [ ] If applicable, have any UI implementations been reviewed by the UX team?

### Security
* [ ] If applicable, has Security reviewed this code?
* [ ] If applicable, is a 3PP request submitted?

### CODE REVIEW

File: cartridges/plugin_pd_image_editor/cartridge/client/default/js/helpers/imageHelperScripts.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+var { elementGenerator } = require('./helperDOM');
+
+function passImageFromTemplateToPDIframe() {
```
Comment (kalipiev) [inline cartridges/plugin_pd_image_editor/cartridge/client/default/js/helpers/imageHelperScripts.js:4]: Mixing var and constants ?

File: cartridges/plugin_pd_image_editor/cartridge/templates/default/experience/components/commerce_assets/customHeroImage.isml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<isscript>
+    var assets = require('*/cartridge/scripts/assets.js');
+    assets.addJs('/js/imageHelpersScripst.js');
+</isscript>
+
+<div>
+    <img class="img-pins" src="${pdict.imgUrl}" alt="${pdict.imageAlt}">
```
Comment (kalipiev) [inline cartridges/plugin_pd_image_editor/cartridge/templates/default/experience/components/commerce_assets/customHeroImage.isml]: Please add an option if this image to be loaded lazy or not.

File: cartridges/plugin_pd_image_editor/cartridge/client/default/js/helpers/imageHelperScripts.js (hunk 1 above)
Comment (VelislavaC4Nexus) [reply cartridges/plugin_pd_image_editor/cartridge/client/default/js/helpers/imageHelperScripts.js:4]: fixed

File: cartridges/plugin_pd_image_editor/cartridge/templates/default/experience/components/commerce_assets/customHeroImage.isml (hunk 2 above)
Comment (VelislavaC4Nexus) [reply cartridges/plugin_pd_image_editor/cartridge/templates/default/experience/components/commerce_assets/customHeroImage.isml]: added lazy loading

**Pull request merged**